### ISSUE TITLE
Regular weight: old style numerals

### ISSUE THREAD

**Issue by eliheuer** [eliheuer/Libre-Plantin#6]
The [regular weight](https://github.com/eliheuer/Libre-Plantin/tree/master/sources/Libre-Plantin-Regular.ufo) needs old-style numerals, using the [Letraset specimen](https://github.com/eliheuer/Libre-Plantin/blob/master/reference/letraset-specimen-110-bold.jpg) as a starting point for the outlines. 

See the numerals section of Design With Font Forge: http://designwithfontforge.com/en-US/Numerals.html

**Comment by StefanPeev** [comment]
@eliheuer And what do you think if we include original look of Plantin numerals as an ONUM (old style figures) through opentype features?

**Comment by StefanPeev** [comment]
Well, I'm preparing now Bulgarian and Serbian/Macedonian Cyrillic Script, after that I'll do the numerals as an ordinary one and as ONUM and just after that I'll give my next Pull Request. I think that with it we could start the first official presentation of the Libre Plantin Regular. Of course after your supervision of the Pull Request and the version.

**Comment by eliheuer** [comment]
Yeah, that is what I meant, I should have been more specific. This is intended as an opentype feature. Thanks for pointing that out! :)
